lib/util.js

```diff
@@ -476,22 +476,22 @@ function formatValue(ctx, value, recurseTimes, ln) {
     noIterator = false;
     if (Array.isArray(value)) {
       // Only set the constructor for non ordinary ("Array [...]") arrays.
-      braces = [`${ctorName === 'Array ' ? '' : ctorName}[`, ']'];
+      braces = [`${prefix === 'Array ' ? '' : prefix}[`, ']'];
       if (value.length === 0 && keyLength === 0)
         return `${braces[0]}]`;
       formatter = formatArray;
     } else if (isSet(value)) {
       if (value.size === 0 && keyLength === 0)
-        return `${ctorName}{}`;
-      braces = [`${ctorName}{`, '}'];
+        return `${prefix}{}`;
+      braces = [`${prefix}{`, '}'];
       formatter = formatSet;
     } else if (isMap(value)) {
       if (value.size === 0 && keyLength === 0)
-        return `${ctorName}{}`;
-      braces = [`${ctorName}{`, '}'];
+        return `${prefix}{}`;
+      braces = [`${prefix}{`, '}'];
       formatter = formatMap;
     } else if (isTypedArray(value)) {
-      braces = [`${ctorName}[`, ']'];
+      braces = [`${prefix}[`, ']'];
       formatter = formatTypedArray;
     } else if (isMapIterator(value)) {
       braces = ['MapIterator {', '}'];
```

Comment:
The `MapIterator` custom designation should not be hardcoded, but `prefix` should instead be used, since `Object.getPrototypeOf(new Map().values())[Symbol.toStringTag] === 'Map Iterator'`. Same applies to `SetIterator` a few lines below.